### PR TITLE
Re-struct some packages

### PR DESCRIPTION
**Pull request by K4L1Ma**

### CODE REVIEW

File: internal/bootstrap/bootstrap.go [hunk 1]
```diff
@@ -1,3 +1,5 @@
+//+build wireinject
```
Comment (hosseio) [inline internal/bootstrap/bootstrap.go:1]: ??

Comment (hosseio) [reply internal/bootstrap/bootstrap.go:1]: This is an example of how to use. You don't use it with this tag, do you?

**Review by hosseio — COMMENTED** [review]
how about `bounded-context-name`?
And I think you are missing to update the `readme.md`

File: internal/bootstrap/bootstrap.go (hunk 1 above)
Comment (K4L1Ma) [reply internal/bootstrap/bootstrap.go:1]: That's Google\Wire tag, I could remove it, I don't mind

this is what I do, these days

```Go
//go:generate wire gen -output_file_prefix bootstrap_ .
```

so my bootstrap is my wire.go and to auto-generate a `bootstrap_wire_gen.go`

Comment (K4L1Ma) [reply internal/bootstrap/bootstrap.go:1]: Added the go:generate to the file

**Pull request merged**